src/lib/blockchain/abis/Session.json

```diff
@@ -119,28 +119,18 @@
       {
         "components": [
           { "internalType": "uint256", "name": "mintId", "type": "uint256" },
-          {
```

Comment:
It gets a bit tricky maintaining 2 versions of the same ABI.

In this case here, it is likely that mainnet will throw an error when using this ABI as it no longer aligns to the actual shape of the data coming back from the blockchain.

We have a couple options here:

1. We maintain 2 Session ABIs in the repo and use based on what network we are on. `SessionMainnet.json` & `SessionTestnet.json`
2. We change the name of the `Recipe` struct in the new contract so that we can keep all of the old ABI structures that are used for minting/recipes on mainnet